spec/forms/steps/case/offence_date_fieldset_form_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+RSpec.describe Steps::Case::OffenceDateFieldsetForm do
+  let(:arguments) { {
+    crime_application: crime_application,
+    record: OffenceDate.new(date: "02, 07, 2020")
```

Comment:
A suggestion, it can be simplified to just:

```ruby
id: record_id
```

and then in a let:

```ruby
let(:record_id) { '12345' }
```

Because in reality this ID comes from the params, not from the record instance. It also allows for simpler tests.